Ports/RISC-V/RV32/GCC/os_cpu_a.S

```diff
@@ -78,27 +83,38 @@
 
 OSStartHighRdy:
 # Disable interrupts
+    # load immediately "RISCV_MSTATUS_MIE" into t0 register
     li     t0, RISCV_MSTATUS_MIE
+    # mstatus = mstatus & t0
```

Comment:
This operation is more like "mstatus &= ~t0" , i.e it clears bits which are set in t0